sdk/loadtestservice/azure-analytics-loadtestservice/swagger/README.md

```diff
@@ -0,0 +1,16 @@
+### Settings
+
+```yaml
+input-file: D:\Project\azure-rest-api-specs\specification\loadtestservice\data-plane\Microsoft.LoadTestService\preview\2022-06-01-preview\loadtestservice.json
```

Comment:
the `input-file` shall be http link like : https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/purview/azure-purview-catalog/swagger#settings